platform/android/java/lib/src/org/godotengine/godot/io/file/FileData.kt

```diff
@@ -59,6 +59,18 @@ internal class FileData(filePath: String, accessFlag: FileAccessFlags) : DataAcc
 			}
 		}
 
+		fun fileLastAccessed(@Suppress("UNUSED_PARAMETER") filepath: String): Long {
+			return 0L
```

Comment:
You can use [`Files.readAttributes(...)`](https://developer.android.com/reference/java/nio/file/Files#readAttributes(java.nio.file.Path,%20java.lang.Class%3CA%3E,%20java.nio.file.LinkOption[])) to retrieve information about the file's last access time.

The method was added in api 26, so you'll need to check the api version before using it, and return `0L` for apis lower than 26 (our min is `21`).